src/defaultSettings.js

```diff
@@ -16,52 +17,66 @@ const defaultSettings = {
             label: 'ALT',
             measure: 'Aminotransferase, alanine (ALT)',
             axis: 'x',
+            imputation: 'data-driven',
             cut: {
-                relative: 3,
+                relative_baseline: 3.8,
```

Comment:
cutoff from the literature?

Reply:
arbitrary for now. Need to ask the docs what the default should be.